### PR TITLE
RDoc-2135-Put-Command-5.0+ changed line 39 in sample file

### PR DESCRIPTION
**Pull request by serezhub**
I only changed one line - line 39 in the sample file

### CODE REVIEW

File: Documentation/5.0/Samples/csharp/Raven.Documentation.Samples/ClientApi/Commands/Documents/Put.cs [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+                var doc = new Category
+                {
+                    Name = "My category",
+                    Description = "My category description"
+                };
+                var docInfo = new DocumentInfo
+                {
+                    Collection = "Categories"
+                };
+                var blittableDoc = session.Advanced.JsonConverter.ToBlittable(doc, docInfo);
```
Comment (reebhub) [inline Documentation/5.0/Samples/csharp/Raven.Documentation.Samples/ClientApi/Commands/Documents/Put.cs:39]: * you updated the sample without updating the method's prototype and explanations about it
* please check in which version the new method replaced the old one, go through the documentation and make sure we document only the correct method.

File: Documentation/5.0/Raven.Documentation.Pages/client-api/commands/documents/put.dotnet.markdown [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+## Syntax
+
+{CODE put_interface@ClientApi\Commands\Documents\Put.cs /}
+
+| Parameters | | |
+| ------------- | ------------- | ----- |
+| **id** | string | unique ID under which document will be stored |
+| **changeVector** | string | Entity changeVector, used for concurrency checks (`null` to skip check) |
+| **document** | BlittableJsonReaderObject | The document to store. You may use `session.Advanced.EntityToBlittable.ConvertEntityToBlittable` to convert your entity to a `BlittableJsonReaderObject` |
```
Comment (reebhub) [inline Documentation/5.0/Raven.Documentation.Pages/client-api/commands/documents/put.dotnet.markdown:13]: you updated the method usage in the sample but left the old explanations here. (and in other copies of the article, e.g. in version 5.2)

**Review by reebhub — CHANGES_REQUESTED** [review]

File: Documentation/5.0/Samples/csharp/Raven.Documentation.Samples/ClientApi/Commands/Documents/Put.cs (hunk 1 above)
Comment (serezhub) [reply Documentation/5.0/Samples/csharp/Raven.Documentation.Samples/ClientApi/Commands/Documents/Put.cs:39]: Added //comments to explain the sample.

We're supposed to change it in 5.0 plus.  I copied this into 5.0 and checked that it bubbles up to 6.0.

File: Documentation/5.0/Raven.Documentation.Pages/client-api/commands/documents/put.dotnet.markdown (hunk 2 above)
Comment (serezhub) [reply Documentation/5.0/Raven.Documentation.Pages/client-api/commands/documents/put.dotnet.markdown:13]: I see.  Good catch. Done

File: Documentation/5.0/Samples/csharp/Raven.Documentation.Samples/ClientApi/Commands/Documents/Put.cs (hunk 1 above)
Comment (reebhub) [reply Documentation/5.0/Samples/csharp/Raven.Documentation.Samples/ClientApi/Commands/Documents/Put.cs:42]: not sure what you changed, the 5.2 explanation remained the same.
please go through the versions as many times as you need and finish it properly.

Comment (serezhub) [reply Documentation/5.0/Samples/csharp/Raven.Documentation.Samples/ClientApi/Commands/Documents/Put.cs:42]: Done.

**Review by reebhub — APPROVED** [review]

**Pull request merged**